internal/pwru/kprobe.go

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2023 Leon Hwang.
```

Comment:
Ah, just spotted this. Could you change to `Copyright 2024 Authors of Cilium`, which includes you?

Reply:
Bad news, no.

Should I change it to `Copyright 2024 Authors of Cilium` or add a more line to add `Copyright 2024 Authors of Cilium`?

Reply:
Let me clarify this with CNCF / Cilium.

Reply:
According to local experts:
> CNCF [advise](https://github.com/cncf/foundation/blob/main/copyright-notices.md#copyright-notices) that copyright should be along the lines of”Copyright 2024 Authors of Cilium”.
> The additional copyright status is discouraged, but if they want to add themselves to [Cilium authors](https://github.com/cilium/cilium/blob/main/AUTHORS) file they’d be entitled to do so.

Reply:
Updated.